components/raftstore-v2/src/operation/command/admin/mod.rs

```diff
@@ -72,7 +75,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             self.propose_conf_change(ctx, req)
         } else {
             // propose other admin command.
-            unimplemented!()
+            self.propose_command(ctx, req)
```

Comment:
Split should have its own customization.

Reply:
why?